screens/FavMoreInfo.tsx

```diff
@@ -2,9 +2,11 @@ import React from 'react';
 import { StyleSheet, View, Text, Image } from 'react-native';
 import styles from '../style/base';
 import PropRow from '../components/PropRow';
+import { Pokemon } from '../api/pokemon';
 
-export default function FavMoreInfo({ route, navigation }) {
-  const { favPokemon } = route.params;
+export default function FavMoreInfo({ route }) {
+  const { favPokemon }: { favPokemon: Pokemon } = route.params;
+  const abilities = favPokemon.abilities.filter(a => !a['is_hidden']);
```

Comment:
nice 😮 I'm curious what `is_hidden` means
Also, I think you could access it like this: `!a.is_hidden`

Reply:
I guess always only one ability is not hidden but I have no idea what's the difference from abilities and moves...